src/test/java/lottogame/domain/lotto/WinningLottoTest.java

```diff
@@ -0,0 +1,38 @@
+package lottogame.domain.lotto;
+
+import lottogame.utils.DuplicateLottoNumberException;
+import lottogame.utils.InvalidBonusBallNumberException;
+import lottogame.utils.InvalidLottoNumberRangeException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class WinningLottoTest {
+
+    @DisplayName("보너스 볼이 로또 번호와 중복되는 경우")
+    @Test
+    void 로또_번호_입력() {
+        assertThatThrownBy(() -> {
+            new WinningLotto(Arrays.asList(1, 2, 3, 4, 5, 6), 6);
+        }).isInstanceOf(DuplicateLottoNumberException.class);
```

Comment:
```suggestion
        assertThatThrownBy(() -> new WinningLotto(Arrays.asList(1, 2, 3, 4, 5, 6), 6))
                .isInstanceOf(DuplicateLottoNumberException.class);
```
위와 같이 중괄호는 생략이 가능합니다 :)